domain/schema/controller.go

```diff
@@ -318,21 +317,51 @@ CREATE TABLE model_list (
 
 func modelMetadataSchema() schema.Patch {
 	return schema.MakePatch(`
+CREATE TABLE model_type (
+    id INT PRIMARY KEY,
+    type TEXT NOT NULL
+);
+
+CREATE UNIQUE INDEX idx_model_type_type
+ON model_type (type);
+
+INSERT INTO model_type VALUES
+    (0, 'none'),
```

Comment:
none is never a valid model type - it must always be set to iaas or caas (or whatever else)
so i don't think we want the 0 value here

Reply:
Done & Done